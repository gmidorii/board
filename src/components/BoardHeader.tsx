import * as React from 'react'

export interface Props {
  title: string
}
export interface State {}

export class BoardHeader extends React.Component<Props, State> {
  constructor(props: Props) {
    super(props)
    this.state = {
      title: 'board title'
    }
  }

  render() {
    return <h3>{this.props.title}</h3>
  }
}
