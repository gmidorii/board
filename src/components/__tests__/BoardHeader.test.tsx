import * as React from 'react'
import * as render from 'react-test-renderer'
import { BoardHeader } from '../BoardHeader'

test('header snapshort', () => {
  const com = render.create(<BoardHeader title="hoge" />)
  expect(com.toJSON()).toMatchSnapshot()
})
