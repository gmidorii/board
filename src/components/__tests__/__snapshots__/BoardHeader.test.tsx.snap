// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header snapshort 1`] = `
<h3>
  hoge
</h3>
`;
